airflow/config_templates/default_airflow.cfg

```diff
@@ -59,6 +59,7 @@ fab_logging_level = WARN
 logging_config_class =
 
 # Log format
+colored_console_log = True
```

Comment:
Do not change the default please.

Reply:
Why not @ashb ? I think that's the whole point to change it by default so that the colours are now enabled whenever you run airflow in terminal (i.e. during development). 

I find it really useful to get the logs coloured by default in console/terminal, especially that pretty much every terminal now support ANSI colors. It saves a lot of time when you are looking through the logs and makes it easier to spot some problems. 

Is there a specific reason why we should not enable it?

Reply:
Making it default to true when connected to a tty I'm fine with, but this config would put ansi codes in the scheduler logs by default when run in "production" too.

Reply:
Note that if some updates are applied - this colouring will only be enabled in console and when there is a terminal allocated at stdout - so pretty much only when you are running in development mode.

Reply:
That would be great then, though a way to force it on even when not TTY (or off when there is a TTY) is probably called for.